Neos.ContentRepository/Classes/Domain/Model/Workspace.php

```diff
@@ -208,9 +208,9 @@ public function setTitle($title)
      * @return string
      * @api
      */
-    public function getDescription()
+    public function getDescription(): string
     {
-        return $this->description;
+        return (string)$this->description;
```

Comment:
The typecast is added to handle `null` in existing data? If so, what about `$title` in `getTitle()`?

Reply:
Title was not configured to be nullable. So I assumed it cannot be null.